options/cf_options.h

```diff
@@ -271,6 +270,7 @@ struct MutableCFOptions {
   Temperature bottommost_temperature;
 
   uint64_t sample_for_compression;
+  std::vector<CompressionType> compression_per_level;
```

Comment:
Minor nit: Can we try to keep the order of the structure elements the same between the ColumnFamilyOptions and the Mutable/Immutable ones?  It makes it easier for comparison/checking purposes.

Reply:
`MutableCFOptions` groups the options by memtable, compaction, blob, misc, we're putting it to misc group with other compression options:
https://github.com/facebook/rocksdb/blob/6cb7d64259110345a86ad48252df8dacd8c623cd/options/cf_options.h#L246-L260